packages/cli/commands/processor.ts

```diff
@@ -20,8 +20,17 @@ async function createRuntimeDependencies(): Promise<
   Record<RUNTIME_IDENTIFIER, any>
 > {
   const { Runtime } = await import("@commontools/runner");
+  const { StorageManager } = await import(
+    "@commontools/runner/storage/cache.deno"
+  );
   const { createBuilder } = await import("@commontools/builder");
-  const builder = createBuilder(new Runtime({ storageUrl: "volatile://" }));
+  const { Identity } = await import("@commontools/identity");
+  const storageManager = StorageManager.emulate({
+    as: await Identity.fromPassphrase("builder"),
```

Comment:
@jsantell should this use the emulate version or connect to a real one with `address`

Reply:
> @jsantell should this use the emulate version or connect to a real one with `address`

eventually, it's only a compiler/invoker at this point